sequence-classification/jupiter_shared/helpers/datasets_helper.py

```diff
@@ -0,0 +1,60 @@
+import json
+import re
+import numpy
+from numpy import genfromtxt
+from sklearn.feature_extraction.text import CountVectorizer
+
+
+def get_train_and_test(data, ratio=0.8):
```

Comment:
Podział na train i test dokonujemy w każdej rundzie w comparatorze. Dzięki temu każdy z klasyfikatorów dostaje te same dane. Używamy do tego: [train_test_split](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html).

Reply:
Wprowadziłem ten podział, bo chciałem się dostosować do konwencji, która była zastosowana w kerasie.
`# (X_train, y_train), (X_test, y_test) = imdb.load_data(num_words=top_words)`
To i tak jest później merge'owane do jedności:
`X = numpy.concatenate((X_train, X_test))
y = numpy.concatenate((y_train, y_test))`

Reply:
Najlepiej by było jakby keras dawał możliwość pobrania tych danych jako X i y, ale nie widzę na stronie (https://keras.io/datasets/#imdb-movie-reviews-sentiment-classification) nawet możliwości wybrania proporcji zbiorów train i test.
My potrzebujemy X i y razem, bo dopiero później w rundach losujemy te podziały na train i test, dlatego lepiej jeśli zbiory danych, które sami dołączamy będą po prostu X i y.
Jak już będzie więcej zbiorów danych to przy okazji można by też opakować zbiór imdb z kerasa w jakąś funkcję, co by można go łatwo pobierać do takiej formy jaką chcemy